__tests__/components/HeaderSystemMenu.test.js

```diff
@@ -23,51 +23,144 @@ SOFTWARE.
 import { shallow } from 'enzyme';
 import HeaderSystemMenu from '../../src/header/HeaderSystemMenu';
 import setLanguage from '../../src/functions/setLanguage';
+import { useMenuHandler } from '@bluecateng/pelagos';
 
 jest.unmock('../../src/header/HeaderSystemMenu');
 
 jest.mock('../../src/hooks/usePlatformData', () =>
     jest
         .fn()
         .mockReturnValueOnce({
-            data: { user: { permissions: { download_logs: false } } },
+            data: {
+                user: {
+                    permissions: { download_logs: false, view_logs: false },
+                },
+            },
         })
         .mockReturnValueOnce({
-            data: { user: { permissions: { download_logs: true } } },
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
         })
         .mockReturnValueOnce({
-            data: { user: { permissions: { download_logs: true } } },
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
+        })
+        .mockReturnValueOnce({
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
+        })
+        .mockReturnValueOnce({
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
+        })
+        .mockReturnValueOnce({
+            data: {
+                user: { permissions: { download_logs: true, view_logs: true } },
+            },
         }),
 );
 
+jest.mock('@bluecateng/pelagos');
+
 describe('HeaderSystemMenu', () => {
     describe('Rendering', () => {
         it('Render HeaderSystemMenu component with default props', () => {
+            useMenuHandler.mockReturnValueOnce({
+                expanded: true,
+                buttonProps: null,
+                menuProps: null,
+                guardProps: null,
+            });
             const wrapper = shallow(
-                <HeaderSystemMenu canDownloadLogs={false} />,
+                <HeaderSystemMenu
+                    canDownloadLogs={false}
```

Comment:
I think you missed to remove these props too.

Reply:
fixed